arangod/VocBase/KeyGenerator.h

```diff
@@ -57,25 +44,20 @@ class KeyGenerator {
   /// @brief destroy the generator
   virtual ~KeyGenerator() = default;
 
- public:
-  /// @brief get the generator type from VelocyPack
-  static GeneratorType generatorType(arangodb::velocypack::Slice const&);
-
   /// @brief check if the specified key generator in the VelocyPack
   /// can be used in this setup (cluster / single-server)
   static bool canUseType(arangodb::velocypack::Slice const&);
 
   /// @brief create a key generator based on the options specified
   static KeyGenerator* factory(arangodb::velocypack::Slice const&);
 
- public:
   virtual bool trackKeys() const = 0;
```

Comment:
could it be that this method is only called in MMFiles ?

Reply:
seems like rocksdb just calls it https://github.com/arangodb/arangodb/blob/devel/arangod/RocksDBEngine/RocksDBCollection.cpp#L1842